cmd/harmony/flags.go

```diff
@@ -993,6 +996,10 @@ var (
 		Usage:     "logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail",
 		DefValue:  defaultConfig.Log.Verbosity,
 	}
+	verbosePrintsFlag = cli.StringSliceFlag{
+		Name:  "verbose-prints",
+		Usage: "verbose prints, available options: config",
```

Comment:
> 1. This config print is default to false. Is this what we expect?

@LeoHChen Would you please confirm this?

> 2. The usage description is way too vague IMO. Could you elaborate a little bit?

fixed in the latest commits
https://github.com/harmony-one/harmony/blob/b87f17f0effdcb2e186251e96df95390c1929fd1/cmd/harmony/flags.go#L999-L1003

> 3. Would it be better if we can add a default value to this flag?

it defaults to print nothing, please see https://github.com/harmony-one/harmony/pull/3734#discussion_r648405055

> 4. Do you think it would be better to aggregate log flags with a prefix `log.verbose-prints`?

fixed in the latest commits